job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -76,23 +76,36 @@ BEGIN
                job.name,
                job.description,
                job.data,
-               to_char(job.create_date, 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"'),
-               to_char(job.last_update_date, 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"'),
+               job.create_date,
+               job.last_update_date,
                job.status,
                job.percentage_complete,
                job.failure_details,
                CAST('WORKER' AS CHAR(6)) AS actionType,
                lbl.label,
                lbl.value
+        FROM
+        (SELECT
+               job.partition_id,
+               job.job_id,
+               job.name,
+               job.description,
+               job.data,
+               to_char(job.create_date, 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"') as create_date,
+               to_char(job.last_update_date, 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"') as last_update_date,
+               job.status,
+               job.percentage_complete,
+               job.failure_details,
+               CAST('WORKER' AS CHAR(6)) AS actionType
         FROM job
-        LEFT JOIN public.label lbl ON lbl.partition_id = job.partition_id AND lbl.job_id = job.job_id
-        $q$;
 
+        $q$;
 
     IF in_labels IS NOT NULL AND ARRAY_LENGTH(in_labels, 1) > 0 THEN
         sql := sql || ' INNER JOIN public.label lbl_filter ON lbl_filter.partition_id = job.partition_id '
```

Comment:
I notice you've changed this but not resubmitted it for review yet.  I'm not sure if that was just a mistake or if you were planning to have another look at it but I'll just plough on in anyway and tell you this still isn't right.  (Hint: You should be thinking about using a Semi-Join here rather than an Inner-Join.)